VetClinicPublic/src/VetClinicPublic/App_Start/Message.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Data.SqlClient;
+using System.IO;
+
+namespace VetClinicPublic.Web.AppStart
+{
+    public class Message
```

Comment:
I'll update this to use a different messaging solution, but it probably will come out of AppStart.